software/libexpat.md

```
@@ -0,0 +1,32 @@
+For example, our default browser [Firefox](./firefox), [QGIS](./qgis), [WinSCP](winscp), [Audacity](./audacity), and Python
+But it’s also used in proprietary software, such as our [Web Application Firewall](https://my.f5.com/manage/s/article/K000158115).
+
+
+## Open Source Sabbatical
+
+In the second half of 2026, we will be offering an [Open Source Sabbatical](../sabbatical) to the maintainer of expat.
+In addition to general software maintenance and further development, the following priorities will be addressed:
+
+* Fix [already known and unfixed security vulnerabilities](https://github.com/libexpat/libexpat/issues/1160).
+  * Addressing additional security vulnerabilities reported as part of the current flood of security analyses.
+* [Support for XML 1.0r5/1.1 ](https://github.com/libexpat/libexpat/issues/171).
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside link text

(MD039, no-space-in-links)